src/main/config/config.c

```diff
@@ -506,8 +506,20 @@ static void validateAndFixConfig(void)
     }
 
 #if defined(USE_DSHOT_TELEMETRY)
-    if ((!configuredMotorProtocolDshot || (motorConfig()->dev.useDshotBitbang == DSHOT_BITBANG_OFF && motorConfig()->dev.useBurstDshot == DSHOT_DMAR_ON) || systemConfig()->schedulerOptimizeRate == SCHEDULER_OPTIMIZE_RATE_OFF)
-        && motorConfig()->dev.useDshotTelemetry) {
+    bool nChannelTimerUsed = false;
+    for (unsigned i = 0; i < getMotorCount(); i++) {
```

Comment:
This caught my attention. `unsigned` is a modifier which can apply to any integral type ( char , short , int , long , etc.) but on its own it is identical to `unsigned int` . There is no difference. unsigned and unsigned int are both synonyms for the same type (the unsigned version of the int type).